### ISSUE TITLE
support python 3 ?

### ISSUE THREAD

**Issue by vkingnew** [MLEveryday/100-Days-Of-ML-Code#88]
whether support python 3.0? like 3.6+ version,eg,3.6,3.7

**Comment by zhyongquan** [comment]
yes

**Comment by ThisIszas** [comment]
Yep, If you have difficult in reading Chinese I'd suggest you check the English version at https://github.com/Avik-Jain/100-Days-Of-ML-Code
